source/docs/articles/drupal/cron.md

```diff
@@ -5,7 +5,7 @@ category:
   - developing
 keywords: drupal, cron, cron execution, run cron,
 ---
-Cron will always run unless all jobs are specifically set to 'Off' via Elysia or Ultimate cron modules. Cron will also not run via Drush if a cron key is set with Elysia.
+Cron will always run unless all jobs are specifically set to 'Off' via Elysia or Ultimate Cron modules. Cron will also not run via Drush if a Cron key is set with Elysia.
 
```

Comment:
I googled it and it varies per site. I treated it as a pronoun in other docs, like we do with Git, unless it's part of a code snippet.